src/FacadeAccountCreation/FacadeAccountCreation.Core/Models/User/OrganisationDetailModel.cs

```diff
@@ -15,4 +15,8 @@ public class OrganisationDetailModel
     public string OrganisationNumber { get; set; }
 
     public int? NationId { get; set; }
+
+    public string OrgAddress { get; set; }
```

Comment:
Rename to OrganisationAddress - we are not under any need to abbreviate property names, so let's ensure they are clear.